### ISSUE TITLE
Doesn`t work on MacOS Sonoma 

### ISSUE THREAD

**Issue by kirilakikiev** [minecraft-linux/mcpelauncher-manifest#894]
<img width="1163" alt="image" src="https://github.com/minecraft-linux/mcpelauncher-manifest/assets/146418733/c9ecf0e3-6b39-4745-8ff6-324d07392697">

Crashing every time when I try launch
Before update all worked fine..

**Comment by DBeitniz** [comment]
Yeah, I already posted this bug previously, if we have a lot people complaining, maybe we will be able to get it working on Sonoma, second of all, just rollback you Mac to 13.4 if you really need to use it :)

**Comment by kirilakikiev** [comment]
> rollback you Mac to 13.4 if you really need to use it :)

If I rollback to 13.4, I will lose all my data) and this is not the best outcome) are there any alternatives to solve this problem? Just after installing Sonoma, two launcher updates arrived, and I updated with the hope that it will eventually work

**Comment by kirilakikiev** [comment]
> rollback you Mac to 13.4 if you really need to use it :)

If I rollback to 13.4, I will lose all my data) and this is not the best outcome) are there any alternatives to solve this problem? Just after installing Sonoma, two launcher updates arrived, and I updated with the hope that it will eventually work

**Comment by ChristopherHX** [comment]
Apply all changes described here https://github.com/minecraft-linux/mcpelauncher-manifest/discussions/819#discussioncomment-7139024

and it works again on m1 macs on sonoma

**Comment by ChristopherHX** [comment]
apple updates are not friendly to opensource software

- macOS 10.15 disabled x86 execution (1+ year of lack of support, the game itself was not 64bit ready on android)
- macOS 14  **broke rosetta 2**, the m1 port is pretty new and highly experimental due to me have no m1 mac and don't test anything

macOS 14 x86_64 is not affected by this problem, so why did apple break rosetta 2 only?

**Comment by kirilakikiev** [comment]
> sonoma

Thank you, it worked for me!!!

**Comment by kirilakikiev** [comment]
> apple updates are not friendly to opensource software
> 
> - macOS 10.15 disabled x86 execution (1+ year of lack of support, the game itself was not 64bit ready on android)
> - macOS 14  **broke rosetta 2**, the m1 port is pretty new and highly experimental due to me have no m1 mac and don't test anything
> 
> macOS 14 x86_64 is not affected by this problem, so why did apple break rosetta 2 only?

If this helps, I launched the game, it works stably on M1 (macbook pro 2020), the performance is good. The only bug that I noticed (it was also in previous versions) when installing the resource pack, it is set to but after re-entering the game, the game freezes on the initial screen without interface buttons, etc., because of this, you have to completely delete the resource pack from the folder, and restart the game, and re-install the resource pack again


Once again, thank you for your help and for developing such a good product.

**Comment by kirilakikiev** [comment]
> apple updates are not friendly to opensource software
> 
> - macOS 10.15 disabled x86 execution (1+ year of lack of support, the game itself was not 64bit ready on android)
> - macOS 14  **broke rosetta 2**, the m1 port is pretty new and highly experimental due to me have no m1 mac and don't test anything
> 
> macOS 14 x86_64 is not affected by this problem, so why did apple break rosetta 2 only?

If this helps, I launched the game, it works stably, the randomness is good. The only bug that I noticed (it was also in previous versions) when installing the resource pack, it is set to but after re-entering the game, the game freezes on the initial screen without interface buttons, etc., because of this, you have to completely delete the resource pack from the folder, and restart the game, and re-install the resource pack again

**Comment by iMonZ** [comment]
> apple updates are not friendly to opensource software
> 
> * macOS 10.15 disabled x86 execution (1+ year of lack of support, the game itself was not 64bit ready on android)
> * macOS 14  **broke rosetta 2**, the m1 port is pretty new and highly experimental due to me have no m1 mac and don't test anything
> 
> macOS 14 x86_64 is not affected by this problem, so why did apple break rosetta 2 only?

Sorry, but those things do not have anything to do with not being open source friendly.
Actually, the opposite is happening. Open Source Projects can recompile faster to a new architecture. 
And the drop of 32bit software had its own reason. But before 'hating' on a companies decision without informing yourself why those decisions were made.
Actually, Windows is the only major operating system to push 32bit support. Linux distros, Android etc. are actually following Apple slowly.
